c2rust-analyze/src/rewrite/mod.rs

```diff
@@ -42,6 +42,12 @@ pub use self::statics::gen_static_rewrites;
 pub use self::ty::dump_rewritten_local_tys;
 pub use self::ty::gen_ty_rewrites;
 
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub enum LifetimeType {
```

Comment:
Bikeshed: `LifetimeName` rather than `LifetimeType` to avoid confusion with actual types.

Reply:
eec91a8e